commit/factory.go

```diff
@@ -66,6 +67,7 @@ type PluginFactory struct {
 
 func NewPluginFactory(
 	lggr logger.Logger,
+	donID uint32,
```

Comment:
do we need a type alias for `uint32`? e.g. `type DonID uint32`

Reply:
Sure, can do a type alias.